src/sentry/integrations/jira/integration.py

```diff
@@ -296,7 +296,10 @@ def create_comment(self, issue_id, user_id, comment):
         return self.get_client().create_comment(issue_id, quoted_comment)
 
     def search_issues(self, query):
-        return self.get_client().search_issues(query)
+        try:
+            return self.get_client().search_issues(query)
+        except Exception as e:
+            self.raise_error(e)
```

Comment:
Should this exception handling go in `search_issues` instead? Just so any other code that uses the client also gets the benefits of this?

Reply:
I think the way we've done things, the client methods are expected to raise `ApiError` (or subclasses of that), which the integration classes are responsible for handling / displaying to user where appropriate. So I don't really think this call belongs in the client's `search_issues` method.

What might be slightly more correct is only handling `ApiError` here rather than all exceptions.